core/controllers/suggestion.py

```diff
@@ -478,18 +478,32 @@ def _construct_exploration_suggestions(suggestions):
     Returns:
         list(dict). List of suggestion dicts with an additional
         exploration_content_html field representing the target
-        exploration's current content.
+        exploration's current content. Suggestions with the target
```

Comment:
The last sentence is unclear. Consider changing it to: 

If the given suggestion refers to an invalid content ID in the current exploration (this can happen if that content was deleted after the suggestion was made), the corresponding suggestion dict will be omitted from the return value.